src/jspdf.js

```diff
@@ -4935,6 +4935,19 @@ function jsPDF(options) {
   };
 
   var lineWidth = options.lineWidth || 0.200025; // 2mm
+  /**
+   * Gets the line width, default: 0.200025.
+   *
+   * @function
+   * @instance
+   * @returns {number} lineHeightFactor
+   * @memberof jsPDF#
+   * @name getLineWidth
+   */
+  var getLineWidth = (API.__private__.getLineWidth = API.getLineWidth = function() {
+    return out(hpf(scale(lineWidth)) + " w");
```

Comment:
`getLineWidth` shouldn't write anything to the document. Just return `lineWidth`.